coderd/scopes_catalog.go

```diff
@@ -0,0 +1,21 @@
+package coderd
+
+import (
+	"net/http"
+
+	"github.com/coder/coder/v2/coderd/httpapi"
+	"github.com/coder/coder/v2/coderd/rbac"
+)
+
+// listExternalScopes returns the curated list of API key scopes (resource:action)
+// requestable via the API.
+//
+// @Summary List API key scopes
+// @ID list-api-key-scopes
+// @Tags Authorization
+// @Produce json
+// @Success 200 {array} string
+// @Router /auth/scopes [get]
+func (*API) listExternalScopes(rw http.ResponseWriter, r *http.Request) {
+	httpapi.Write(r.Context(), rw, http.StatusOK, rbac.ExternalScopeNames())
```

Comment:
I like to nest these kinds of responses into a json struct in case we need to append more information in the future. Just makes the endpoint more flexible long term. Your call.

(make this a real type, not anonymous if you do this)
```suggestion
	httpapi.Write(r.Context(), rw, http.StatusOK, struct {
		External []string `json:"external"`
	}{
		External: rbac.ExternalScopeNames(),
	})
```